src/bot/commands/AvatarCommand.ts

```diff
@@ -33,13 +33,14 @@ export default class AvatarCommand implements UserCommand {
       return;
     }
 
-    if (message.attachments.size !== 1) {
+    const attachment = message.attachments.first();
+    if (message.attachments.size !== 1 || !attachment) {
```

Comment:
You can remove the check for `message.attachments.size !== 1` since you are already getting  the first element out of it.